pkgs/tools/system/undaemonize/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation {
+  name = "undaemonize";
+  src = fetchFromGitHub {
+    repo = "undaemonize";
+    owner = "nickstenning";
+    rev = "master";
```

Comment:
add a concrete and no branch here. Otherwise it will break if new commits are added.